PURL-TYPES.rst

```diff
@@ -258,15 +258,13 @@ golang
 
 - There is no default package repository: this is implied in the namespace
   using the ``go get`` command conventions.
-- The ``namespace`` and `name` must be lowercased.
```

Comment:
where is the namespace/name spec gone? 
see https://github.com/package-url/purl-spec/issues/308